Core/include/Acts/Utilities/BinAdjustment.hpp

```diff
@@ -27,9 +27,13 @@ namespace Acts {
 /// @param rBounds the Radial bounds to adjust to
 ///
 /// @return new updated BinUtiltiy
-BinUtility adjustBinUtility(const BinUtility& bu, const RadialBounds& rBounds) {
+BinUtility adjustBinUtility(const BinUtility& bu, const RadialBounds& rBounds,
+                            const Transform3D& transform) {
   // Default constructor
   BinUtility uBinUtil;
+  if (transform.matrix() != Transform3D::Identity().matrix()) {
+    uBinUtil = BinUtility(std::make_shared<const Transform3D>(transform));
+  }
```

Comment:
One could just test
`transform == Transform3D::Identity()` or even isApprox(), right?

Reply:
I replace the != with isApprox